packages/@react-aria/toast/src/useToast.ts

```diff
@@ -87,15 +91,35 @@ export function useToast<T>(props: AriaToastProps<T>, state: ToastState<T>, ref:
   let descriptionId = useSlotId();
   let stringFormatter = useLocalizedStringFormatter(intlMessages, '@react-aria/toast');
 
+  let onClosePress = () => {
```

Comment:
We're still not handling the case where shouldCloseOnAction is clicked instead of the close button